2024-07-28 11:21:58,242 INFO    MainThread:14923 [wandb_setup.py:_flush():76] Current SDK version is 0.17.3
2024-07-28 11:21:58,242 INFO    MainThread:14923 [wandb_setup.py:_flush():76] Configure stats pid to 14923
2024-07-28 11:21:58,242 INFO    MainThread:14923 [wandb_setup.py:_flush():76] Loading settings from /home/job/.config/wandb/settings
2024-07-28 11:21:58,242 INFO    MainThread:14923 [wandb_setup.py:_flush():76] Loading settings from /home/job/Documents/Thesis/code/experiments/wandb/settings
2024-07-28 11:21:58,242 INFO    MainThread:14923 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-07-28 11:21:58,242 INFO    MainThread:14923 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-07-28 11:21:58,242 INFO    MainThread:14923 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-07-28 11:21:58,242 INFO    MainThread:14923 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-07-28 11:21:58,242 WARNING MainThread:14923 [wandb_setup.py:_flush():76] interrupted
Traceback (most recent call last):
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/pytorch_lightning/trainer/call.py", line 44, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 579, in _fit_impl
    self._run(model, ckpt_path=ckpt_path)
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 948, in _run
    call._call_setup_hook(self)  # allow user to set up LightningModule in accelerator environment
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/pytorch_lightning/trainer/call.py", line 88, in _call_setup_hook
    if hasattr(logger, "experiment"):
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/lightning_fabric/loggers/logger.py", line 118, in experiment
    return fn(self)
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/pytorch_lightning/loggers/wandb.py", line 406, in experiment
    self._experiment = wandb.init(**self._wandb_init)
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1170, in init
    wi.setup(kwargs)
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 300, in setup
    wandb_login._login(
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/wandb/sdk/wandb_login.py", line 331, in _login
    logged_in = wlogin.login()
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/wandb/sdk/wandb_login.py", line 180, in login
    self._print_logged_in_message()
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/wandb/sdk/wandb_login.py", line 186, in _print_logged_in_message
    username = self._wl._get_username()
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/wandb/sdk/wandb_setup.py", line 194, in _get_username
    self._load_viewer()
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/wandb/sdk/wandb_setup.py", line 216, in _load_viewer
    s.query_with_timeout()
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/wandb/sdk/lib/server.py", line 35, in query_with_timeout
    viewer_tuple, viewer_thread = async_viewer()
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/site-packages/wandb/util.py", line 1191, in wrapper
    result = q.get(True, timeout)
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/queue.py", line 180, in get
    self.not_empty.wait(remaining)
  File "/home/job/miniconda3/envs/LTN/lib/python3.9/threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-07-28 11:21:58,251 INFO    MainThread:14923 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-07-28 11:21:58,252 INFO    MainThread:14923 [wandb_init.py:_log_setup():520] Logging user logs to ./wandb/run-20240728_112158-f1qmz9s6/logs/debug.log
2024-07-28 11:21:58,252 INFO    MainThread:14923 [wandb_init.py:_log_setup():521] Logging internal logs to ./wandb/run-20240728_112158-f1qmz9s6/logs/debug-internal.log
2024-07-28 11:21:58,252 INFO    MainThread:14923 [wandb_init.py:_jupyter_setup():466] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x773f526b78e0>
2024-07-28 11:21:58,252 INFO    MainThread:14923 [wandb_init.py:init():560] calling init triggers
2024-07-28 11:21:58,252 INFO    MainThread:14923 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {}
2024-07-28 11:21:58,253 INFO    MainThread:14923 [wandb_init.py:init():610] starting backend
2024-07-28 11:21:58,253 INFO    MainThread:14923 [wandb_init.py:init():614] setting up manager
2024-07-28 11:21:58,255 INFO    MainThread:14923 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-07-28 11:21:58,257 INFO    MainThread:14923 [wandb_init.py:init():622] backend started and connected
2024-07-28 11:21:58,274 INFO    MainThread:14923 [wandb_run.py:_label_probe_notebook():1334] probe notebook
2024-07-28 11:21:58,275 INFO    MainThread:14923 [wandb_run.py:_label_probe_notebook():1344] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-07-28 11:21:58,275 INFO    MainThread:14923 [wandb_init.py:init():711] updated telemetry
2024-07-28 11:21:58,327 INFO    MainThread:14923 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2024-07-28 11:21:59,174 INFO    MainThread:14923 [wandb_run.py:_on_init():2402] communicating current version
2024-07-28 11:21:59,372 INFO    MainThread:14923 [wandb_run.py:_on_init():2411] got version response upgrade_message: "wandb version 0.17.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-07-28 11:21:59,372 INFO    MainThread:14923 [wandb_init.py:init():795] starting run threads in backend
2024-07-28 11:22:04,724 INFO    MainThread:14923 [wandb_run.py:_console_start():2380] atexit reg
2024-07-28 11:22:04,725 INFO    MainThread:14923 [wandb_run.py:_redirect():2235] redirect: wrap_raw
2024-07-28 11:22:04,725 INFO    MainThread:14923 [wandb_run.py:_redirect():2300] Wrapping output streams.
2024-07-28 11:22:04,725 INFO    MainThread:14923 [wandb_run.py:_redirect():2325] Redirects installed.
2024-07-28 11:22:04,728 INFO    MainThread:14923 [wandb_init.py:init():838] run started, returning control to user process
2024-07-28 11:22:04,729 INFO    MainThread:14923 [wandb_run.py:_finish():2109] finishing run jjhlgraber-University of Amsterdam/first_project/f1qmz9s6
2024-07-28 11:22:04,730 ERROR   MainThread:14923 [jupyter.py:save_history():436] Run pip install nbformat to save notebook history
2024-07-28 11:22:04,730 INFO    MainThread:14923 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2024-07-28 11:22:04,730 INFO    MainThread:14923 [wandb_init.py:_jupyter_teardown():448] cleaning up jupyter logic
2024-07-28 11:22:04,730 INFO    MainThread:14923 [wandb_run.py:_atexit_cleanup():2349] got exitcode: 0
2024-07-28 11:22:04,730 INFO    MainThread:14923 [wandb_run.py:_restore():2332] restore
2024-07-28 11:22:04,731 INFO    MainThread:14923 [wandb_run.py:_restore():2338] restore done
2024-07-28 11:22:08,390 INFO    MainThread:14923 [wandb_run.py:_footer_history_summary_info():3991] rendering history
2024-07-28 11:22:08,390 INFO    MainThread:14923 [wandb_run.py:_footer_history_summary_info():4023] rendering summary
2024-07-28 11:22:08,398 INFO    MainThread:14923 [wandb_run.py:_footer_sync_info():3950] logging synced files
